docs/reference/management_commands.md

```diff
@@ -52,12 +52,31 @@ This command deletes old revisions which are not in moderation, live, approved t
 revision. If the `days` argument is supplied, only revisions older than the specified number of
 days will be deleted.
 
-If the `pages` argument is supplied, only revisions of page models will be deleted. If the `non-pages` argument is supplied, only revisions of non-page models will be deleted. If both or neither arguments are supplied, revisions of all models will be deleted.
+If the `pages` argument is supplied, only revisions of page models will be deleted. If the `non-pages` argument is supplied, only revisions of non-page models will be deleted. If both or neither arguments are supplied, revisions of all models will be deleted. 
+If deletion of a revision is not desirable, mark `Revision` with `on_delete=models.PROTECT`. 
```

Comment:
It seems to read better if we add this before the `pages` and `non-pages` option, e.g.

````suggestion
To prevent deleting important revisions when they become stale, you can refer to such revisions in a model using a `ForeignKey` with {attr}`on_delete=models.PROTECT <django.db.models.PROTECT>`.

```{versionadded} 5.2
Support for respecting `on_delete=models.PROTECT` is added.
```

If the `pages` argument is supplied, only revisions of page models will be deleted. If the `non-pages` argument is supplied, only revisions of non-page models will be deleted. If both or neither arguments are supplied, revisions of all models will be deleted.
````

Reply:
I'm 💯 behind this suggestion!